converters/json2glm.py

```diff
@@ -18,7 +18,7 @@ def clock_glm(data, filename_out):
 		fw.write(start_str)
 		fw.write(stop_str)
 		fw.write(end_str)
-	return data, filename_out
+	return True
```

Comment:
All these returns are never used. Why have them?

Reply:
Removed the return statements